lib/organize_letters.rb

```diff
@@ -0,0 +1,36 @@
+require_relative './ds/heap'
+
+def data = Struct.new(:value, :frequency)
+
+def organize_letters(string)
+  letters = Hash.new(0)
+  queue = Heap.new
+  string_size = string.size
+  new_string = ""
+  
+  string.each_char { |char| letters[char] += 1 }
```

Comment:
![image](https://github.com/ImMPrada/short-ds-challenges/assets/26731448/82fa2ce8-3b8f-4fa3-89ec-bde66eecf429)

computing the frequency of each leter